OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/CreateKeyFinalFragment.java

```diff
@@ -300,7 +300,9 @@ private static SaveKeyringParcel createDefaultSaveKeyringParcel(CreateKeyActivit
 
             if(createKeyActivity.mPassphrase != null) {
                 saveKeyringParcel.setNewUnlock(new ChangeUnlockParcel(createKeyActivity.mPassphrase));
-            }
+            } else {
+				saveKeyringParcel.setNewUnlock(null);
+			}
```

Comment:
Sorry about that. The tab snuck in when I was using meld for conflict resolution. I'll be more careful in the future.